### DISCUSSION TITLE
Display latest commit date and time for all project' forks

### DISCUSSION THREAD

**Discussion by superbonaci** [community/community#32820] [Other Feature Feedback, Questions, & Ideas]
Hi support, would like that forks list for a project include latest commit date and time, example:

https://github.com/DoTheEvo/ANGRYsearch/network/members

There are 78 forks of this project, and would like to be able to sort them by latest commit date, so I can know if any of the forks has any commit later than the latest commit of the original project which all were forked from.

**Comment by DenverCoder1** [comment]
There's a site called [useful-forks.github.io](https://github.com/useful-forks/useful-forks.github.io) that can help as a temporary solution to this problem.

It will show you the number of stars, forks, commits ahead, and commits behind, as well as the last commit date for each relevant fork.

https://useful-forks.github.io/?repo=DoTheEvo%2FANGRYsearch

![image](https://user-images.githubusercontent.com/20955511/189790291-a5b1b2f8-fa20-4c3a-9de1-e5ff0c7074c8.png)

There is also a third-party browser extension called [Refined GitHub](https://github.com/refined-github/refined-github) that will create a button on the forks page linking to Useful Forks.

![image](https://user-images.githubusercontent.com/20955511/189790569-c3105708-86e2-4506-8900-e5eeb320db58.png)

**Reply by superbonaci** [reply]
It has GitHub API rate limit.
